Fireworks/Geometry/python/dumpRecoGeometry_cfg.py

```diff
@@ -35,18 +35,13 @@ def recoGeoLoad(score):
 
     elif score == "2017":
        from Configuration.AlCa.GlobalTag import GlobalTag
-       process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:upgrade2017', '')
+       process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:mc', '')
        process.load('Configuration.Geometry.GeometryExtended2017Reco_cff')
-       # Automatic addition of the customisation function from SLHCUpgradeSimulations.Configuration.combinedCustoms
-       #from SLHCUpgradeSimulations.Configuration.combinedCustoms import cust_2017 
-       #process.load("SLHCUpgradeSimulations.Configuration.combinedCustoms.cust_2017")
-      # process = cust_2017(process)
 
     elif  score == "2019":
-      from Configuration.AlCa.GlobalTag import GlobalTag
-      process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:upgrade2019', '')
+      from Configuration.AlCa.autoCond import autoCond
+      process.GlobalTag.globaltag = autoCond['mc']
```

Comment:
@jshlee - this will pickup run1 geometry and not 2019. Please, discard this change.